packages/combo-box/test/selecting-items.test.js

```diff
@@ -234,7 +234,7 @@ describe('selecting items', () => {
 
       comboBox.open();
       comboBox.value = 'foo';
-      comboBox.close();
+      clickItem(comboBox, 0);
```

Comment:
Should we use outsideClick here as well?

```suggestion
      outsideClick();
```